app.py

```diff
@@ -82,6 +82,8 @@ def playBGM(bgm_filename, volume=1.0):
     pygame.mixer.music.play(loops=-1)
 def stopBGM():
     pygame.mixer.music.stop()
+def stopSound():
+    pygame.mixer.sound.stop()
```

Comment:
Was this tested and verified on a device with sound? Can't accept anything into the RC until the feature was validated as working in it's own branch.

Reply:
No it wasn't If you can test it or maybe figure it out, i would've tested it myself but the one in the back doenst have any sound.

Reply:
I'm going to remove this file and accept the rest of the PR. We can test this fix on the main board or find some speakers for you next time we have class.